src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java

```diff
@@ -782,16 +782,16 @@ public void testWriteAccessRightElement() throws XMLStreamException, IOException
         // given
         DatasetDTO datasetDto = mapObjectFromJsonTestFile("export/dataset-all-defaults.txt", DatasetDTO.class);
         DatasetVersionDTO dto = datasetDto.getDatasetVersion();
-    
+
         // when
         OpenAireExportUtil.writeAccessRightsElement(xmlWriter, dto, null);
         xmlWriter.flush();
-    
+
         //then
         assertEquals("<rightsList>"
                 + "<rights rightsURI=\"info:eu-repo/semantics/closedAccess\"/>"
                 + "<rights rightsURI=\"https://creativecommons.org/publicdomain/zero/1.0/\">"
-                + "CC0 Waiver</rights></rightsList>",
+                + "CC0-1.0</rights></rightsList>",
```

Comment:
This is tricky too. The original code is getting 'CC0 Waiver' because naming the license 'CC0' is triggering a license specific code to then write 'CC0 Waiver' in the terms of use field which is what shows up here. As I mentioned elsewhere, it would be possible to use 'CC0 Waiver' as the name of the license - by default or at an institution - to have that appear wherever 'CC0 1.0' would be. That would make this license the only one with a non-standard name though (all the rest, after correcting the hyphens) are named as they are at Creative Commons.

It's also useful to note that the URL on the line before this was hardcoded in the old code. To get the URL of any license, it needs to be part of the json export that is what most of the exporters use - which is why license goes from being a string to having a object with name/url in the basic json export. (One could also redesign to let the exporters call the LicenseServiceBean to get the URL from the name, but that means making the exporters EJB aware or added the LicenseServiceBean as a parameter, etc.)